### PR TITLE
Add get_chat_onlines method

### PR DESCRIPTION
**Pull request by AndrielFR**
Used to get the number of online users in a specific group.

### CODE REVIEW

File: pyrogram/methods/chats/get_chat_onlines.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+from pyrogram import raw
+from pyrogram.scaffold import Scaffold
+
+
+class GetChatOnlines(Scaffold):
+    async def get_chat_onlines(
+        self,
+        chat_id: Union[int, str]
+    ) -> Optional[int]:
```
Comment (alissonlauffer) [inline pyrogram/methods/chats/get_chat_onlines.py:29]: The function does not return None. Just int or raise an exception.

```suggestion
    ) -> int:
```

File: pyrogram/methods/chats/get_chat_onlines.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#
+#  Pyrogram is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Optional, Union
```
Comment (alissonlauffer) [inline pyrogram/methods/chats/get_chat_onlines.py:19]: Unused import when applying the next suggestion.

```suggestion
from typing import Union
```

**Review by alissonlauffer — CHANGES_REQUESTED** [review]

File: pyrogram/methods/chats/get_chat_onlines.py (hunk 1 above)
Comment (AndrielFR) [reply pyrogram/methods/chats/get_chat_onlines.py:29]: 😳

File: pyrogram/methods/chats/get_chat_onlines.py (hunk 2 above)
Comment (AndrielFR) [reply pyrogram/methods/chats/get_chat_onlines.py:19]: Ok.

**Comment by delivrance** [comment]
`get_chat_online_count` sounds better to me.

**Review by delivrance — CHANGES_REQUESTED** [review]
`get_chat_online_count` sounds better to me.

**Pull request merged**